pyproject.toml

```diff
@@ -29,11 +29,7 @@ transformers = [
 ]
 # chatglm2 custom tokenizer file imports torch, have to add torch dependency for tests
 torch = [
-    'torch @ https://download.pytorch.org/whl/cpu-cxx11-abi/torch-2.0.1%2Bcpu.cxx11.abi-cp38-cp38-linux_x86_64.whl ; sys_platform=="linux" and python_version == "3.8"',
-    'torch @ https://download.pytorch.org/whl/cpu-cxx11-abi/torch-2.0.1%2Bcpu.cxx11.abi-cp39-cp39-linux_x86_64.whl ; sys_platform=="linux" and python_version == "3.9"',
-    'torch @ https://download.pytorch.org/whl/cpu-cxx11-abi/torch-2.0.1%2Bcpu.cxx11.abi-cp310-cp310-linux_x86_64.whl ; sys_platform=="linux" and python_version == "3.10"',
-    'torch @ https://download.pytorch.org/whl/cpu-cxx11-abi/torch-2.0.1%2Bcpu.cxx11.abi-cp311-cp311-linux_x86_64.whl ; sys_platform=="linux" and python_version == "3.11"',
-    'torch ; sys_platform != "linux"'
+    'torch'
```

Comment:
it will download CUDA version

Reply:
we don't use it at all, that is dependency for one of the test models and it also doesn't use it

Reply:
It's used by `transformers`, which we use in product 
https://github.com/openvinotoolkit/openvino_tokenizers/blob/master/python/openvino_tokenizers/hf_parser.py#L19-L20

Reply:
@apaniukov could you comment that?

Reply:
@ilya-lavrenov No, `torch` is an optional dependency of `transformers` that we didn't use. ChatGLM2 custom tokenizer [imports](https://huggingface.co/THUDM/chatglm2-6b/blob/7fabe56db91e085c9c027f56f1c654d137bdba40/tokenization_chatglm.py#L2) torch. Here is the PR to remove it: https://huggingface.co/THUDM/chatglm2-6b/discussions/108, but no response yet.